app/services/tax_rate_finder.rb

```diff
@@ -77,6 +77,6 @@ def find_closest_tax_rates_from_included_tax(amount, included_tax)
     approximation = (included_tax / (amount - included_tax))
     return [] if approximation.infinite? || approximation.zero? || approximation.nan?
 
-    [Spree::TaxRate.order("ABS(amount - #{approximation})").first]
+    [Spree::TaxRate.order(Arel.sql("ABS(amount - #{approximation})")).first]
```

Comment:
What were the errors here? There's a good chance there might be an alternative solution that doesn't require hacks to work around security warnings...

Reply:
Ah I see... https://github.com/openfoodfoundation/openfoodnetwork/issues/7610

Reply:
Okay :+1:

Previously there was a strong recommendation from Rails core that you should try to avoid using Arel directly (as it was meant to be a private API used by ActiveRecord under the hood), but it looks like they're changed that for Rails 6.

Reply:
yes, this is really a workaround to the security restriction  :+1:

Reply:
this is now in code review in #7610